### PR TITLE
Add position:sticky

### PR DESCRIPTION
**Pull request by tristen**

### CODE REVIEW

File: src/layout.css [hunk 1]
```diff
@@ -224,6 +225,10 @@
 .absolute { position: absolute !important; }
 .relative { position: relative !important; }
 .static { position: static !important; }
+.sticky {
+  position: sticky !important;
+  position: -webkit-sticky !important; /* stylelint-disable-line value-no-vendor-prefix */
```
Comment (tristen) [inline src/layout.css]: Safari still requires this. Unless there's a different way to handle adding this, I think disabling the style lint rule here makes sense.

Comment (samanpwbb) [reply src/layout.css]: We use the autoprefixer postCSS plugin so there's no need for prefixes: https://github.com/mapbox/assembly/blob/publisher-staging/scripts/build-css.js#L114-L116

**Review by samanpwbb — APPROVED** [review]

**Review by samanpwbb — APPROVED** [review]
@tristen looks like you have a snapshot to update here.

**Comment by katydecorah** [comment]
Does this class include media query sizes? If not, I think we should consider adding them. The docs team created `.sticky-mm` and `.sticky-mxl` to handle our sticky sidebars on larger screens.

**Comment by tristen** [comment]
> Does this class include media query sizes? If not, I think we should consider adding them. The docs team created `.sticky-mm` and `.sticky-mxl` to handle our sticky sidebars on larger screens.

Ah! good catch. Updating to include those 👍

**Review by katydecorah — APPROVED** [review]
Perfect!

**Pull request merged**